src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1369,16 +1371,8 @@ function build_if_list() {
 		(isset($portValues[ $pconfig[$type .'endport'] ]) ? null : $pconfig[$type .'endport'])
 	))->setHelp('Custom');
 
-
-	if ($type == 'src')
-		$group->setHelp('Specify the source port or port range for this rule. This is '.
-			'usually random and almost never equal to the destination port range (and '.
-			'should usually be <b>any</b>).  The "To" field may be left '.
-			'empty if only filtering a single port.');
-	else
-		$group->setHelp('Specify the destination port or port range for this rule. ' .
-			'The "To" field may be left empty if only filtering a '.
-			'single port.');
+	$group->setHelp("Specify the {$name} port or port range for this rule. " .
```

Comment:
$name should be in lower case to avoid having messages like:
"Specify the Destination port"

Other than that, setHelp() calls are added to .po to be translated, then this message should be prepared for translation using sprintf()